src/integrate/svr_utilities.cuh

```diff
@@ -83,8 +46,10 @@ static double gasdev()
   }
 }
 
-static double gamdev(const int ia)
+static double gamdev(const int ia, std::mt19937& rng)
 {
+  std::uniform_real_distribution<double> rand1(0, 1);
```

Comment:
Would a `static` declaration improve the performance?

Reply:
using static variables might make the function a little bit faster, but any CPU part is not a bottleneck in GPUMD. Using static variables are not good for possible future multi-threaded extension. So I do not use static variables at all.

Reply:
ok, that makes sense :+1: